pkg/auth/providers/common/util.go

```diff
@@ -0,0 +1,40 @@
+package common
```

Comment:
Can this move into a provider specific client file?

Reply:
moved this code to an oidc_client file